Formula/tezos-signer.rb

```diff
@@ -27,6 +27,8 @@ class << self
 
   bottle do
     root_url "https://github.com/serokell/tezos-packaging/releases/download/#{TezosSigner.version}/"
+    sha256 cellar: :any, mojave: "2e08dcff04997a400820216948958b3694e7aa5ef30e5b5647d23bd0b0c39f43"
```

Comment:
It looks like this is incorrect, this hash is the one for `tezos-accuser-009-PsFLoren-v9.5-1.mojave.bottle.tar.gz`.

This one should be:
```suggestion
    sha256 cellar: :any, mojave: "a91d5a33d657267a4bbe00405a323cd3eedac712e8d804ee0d6e1862f62ad9d8"
```